libs/langserve/langserve/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .client import RemoteRunnable
```

Comment:
in langchain we always do absolute imports, dunno if ew care about sticking to that

Reply:
Ah good catch -- I'm used to explicit relative, but will change to absolute to be consistent